src/utils/apiCalls.js

```diff
@@ -23,4 +22,17 @@ export const getUser = async (userInfo, url) => {
     return response
 }
 
+export const postFavorite = async (movieInfo, id) => {
+  const options = {
+    method: 'POST',
+    body: JSON.stringify(movieInfo),
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  }
 
+  const response = await fetch(`http://localhost:3001/api/v1/users/${id}/moviefavorites`, options)
```

Comment:
I would think that since we already have the id of the user in the global state we would want to use that here, vs passing an extra variable. Did yall try to import that user id from state here as well ? Or did it not work?